src/components/form-elements/pages/input-interactive.jsx

```diff
@@ -48,17 +40,14 @@ const Inputs = () => {
 
   return (
     <div>
-      <DocsActiveBlock settings={settings}>
-        <Input placeholder="Simple input" />
-      </DocsActiveBlock>
-      <DocsActiveBlock settings={settings}>
+      <DocsActiveBlock backgroundColor="dark" settings={settings}>
         <Input
-          placeholder="Big wrong input"
+          backgroundColor="none"
+          placeholder="Big input"
```

Comment:
maybe use a `large` word here too? wdyt? instead of `big`

Reply:
sure thing.

Reply:
solved